README.md

```diff
@@ -1028,7 +1028,7 @@ Table of Contents
 **[⬆ back to top](#table-of-contents)**
 
 ## Web Hosting
-
+  * [1mb Hosting](https://1mb.co/) — Easily create a website or blog in just a few clicks. Customize and update your site from anywhere.  
```

Comment:
I see no privacy policy, no T's and C's no link to contact details of those who operate this site.  These are quite basic requirements for a business service.  The point of this list is to mainly show business services that offer a free tier.